public/add-to-storage.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var addAnswersToSessionStorage = function(){
+  var nameInput = document.getElementsByClassName("name-input")[0];
+  var ageInput = document.getElementsByClassName("age-input")[0];
+  var strange = document.getElementById("strange");
+  var happy = document.getElementById("happy");
+  var angry = document.getElementById("angry");
+  var fun = document.getElementById("fun");
+  var boring = document.getElementById("boring");
+  var kind = document.getElementById("kind");
+  var sad = document.getElementById("sad");
+  var happyEmoji = document.getElementsByClassName("emoji")[0];
+  var sadEmoji = document.getElementsByClassName("emoji")[1];
+  var indifferentEmoji = document.getElementsByClassName("emoji")[2];
+  var personality = [];
+
+  addKeyupEvent('name', nameInput);
+  addKeyupEvent('age', ageInput);
+
+  addClickEventArray('personality', strange, 'strange', personality);
+  addClickEventArray('personality', happy, 'happy', personality);
+  addClickEventArray('personality', angry, 'angry', personality);
+  addClickEventArray('personality', fun, 'fun', personality);
+  addClickEventArray('personality', boring, 'boring', personality);
+  addClickEventArray('personality', kind, 'kind', personality);
+  addClickEventArray('personality', sad, 'sad', personality);
+
+  addClickEventSingle('feelings', sadEmoji, 'sad');
+  addClickEventSingle('feelings', happyEmoji, 'happy');
+  addClickEventSingle('feelings', indifferentEmoji, 'indifferent');
+
+  function addKeyupEvent(key, element){
+    if(!element){ return; };
+    element.addEventListener("keyup", function(){
+      addToStorageValue(key, element.value)
+    });
+  };
+
+  function addClickEventSingle(key, element, value){
+    if(!element){ return; };
+    element.addEventListener("click", function(){
+      addToStorageValue(key, value)
```

Comment:
This is how it is saved in local storage:

![screen shot 2017-03-13 at 10 05 17](https://cloud.githubusercontent.com/assets/18164707/23849764/ab09172e-07d4-11e7-9c98-2d95da3deba7.png)

We need to store all of the answers from the questionnaire and this will data will be added to a PDF at the end of the app.